src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -66,7 +63,7 @@
 
     // The configured signature schemes for "signature_algorithms" and
     // "signature_algorithms_cert" extensions
-    List<SignatureScheme>       signatureSchemes;
+    String[]                   signatureSchemes;
```

Comment:
I would keep the typed list (get it from the parameters or system property cache).

Reply:
I see your point.  There is a tricky question that it would be nice to return (SSLParameters.getXXX()) to what it has been set (SSLParameters.setXXX()).  If the typed list get kept, it means that unknown names would filtered out, and the get method cannot return the property values.